node/src/reactor/joiner.rs

```diff
@@ -201,6 +202,10 @@ pub(crate) enum JoinerEvent {
     #[from]
     BeginAddressGossipRequest(BeginGossipRequest<GossipedAddress>),
 
+    /// Contract runtime request.
+    #[from]
+    ContractRuntimeRequest(ContractRuntimeRequest),
```

Comment:
Same as in initializer.

Reply:
This has been changed in https://github.com/casper-network/casper-node/pull/2614/commits/319ff105afcd205c0e142956c778a8742a5a6d1c

Reply:
As discussed, the change is reverted for now in d7f147fe8b0162bb44534ad7554ce01a37caa323 as this will be a part of bigger refactoring.